packages/core/addon/models/bard-request-v2/request.ts

```diff
@@ -347,6 +351,31 @@ export default class RequestFragment extends Fragment.extend(Validations) implem
     this.sorts.pushObject(this.fragmentFactory.createSort(type, source, field, parameters, direction));
   }
 
+  /**
+   * Pushes column cid to end of rollup list, removing dupes.
+   * @param column - dimension column to add a rollup
+   */
+  pushRollupColumn(column: ColumnFragment) {
```

Comment:
What if the `cid` doesn't exist in columns?

Reply:
typescript error.
or you suggesting we should generate one here if it's empty?

Reply:
I was thinking along the lines of asserting the column fragment exists in the `columns`  array

Reply:
we could add that assertion yes.